server/test/routes/oauth.test.js

```diff
@@ -41,3 +42,6 @@ describe('Social Login', () => {
     }).timeout(50000);
   });
 });
+};
+
+export default oAuthTest;
```

Comment:
why are you exporting the test ?

Reply:
I am exporting the test files because of this configuration
`  after((done) => {
    models.sequelize.close();
  done();
  });
`
in test/routes/users.test that was blocking some test files from running. I move the configuration to test/index.test. So exporting the test files and importing them to index.test file will allow us to specify the order the tests should run